script.js

```diff
@@ -11,3 +12,154 @@ document.querySelectorAll('.navlink').forEach((n) => n
+      <li>${item.back}</li>               
+      <li>${item.years}</li>    
+  </ul>
+</div>
+  <p>
+  ${item.descrip}
+  </p>
+  <ul class="skills-card">
+      <li class="skill-1">html</li>
+      <li class="skill-1">css</li>
+      <li class="skill-1">Javascript</li>
+  </ul>
+  <button class="btn" onclick="display(${item.id})">See Project</button>
+</div>   
+</article>
+      
+     <div class="popup-window" id="element-${item.id}">
+      <div class="popup">
+      <div class="pop-head">
+      <h2 class="title-card">${item.title}</h2>
+      <button class="close-button" aria-hidden="true" onclick="undisplay(${item.id})">&times;</button>
+      </div>
+      <div class="header-details">
+            <h4>
+              CANOPY
+            </h4>
+            <ul class="dot">
+              <li >
+                Back End Dev
+              </li>
+              <li>
+                2015
+              </li>
+            </ul>
+          </div>
+          <figure class="popup-image-container">
+          <img class="desktop"src="${item.Image}" alt="tonic project" />
+          <img class="mobile" src="${item.Images}" alt="tonic project" />
+        </figure>
+        
+        <div class="text-section">
+        <p class="mobile">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+        <p class="desktop">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.</P>
+
+        <p class="desktop">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.</P>
+        <div class="button-section">
+        <ul class="skills-card">
+          <li class="skill-1">
+            html
+          </li>
+          <li class="skill-1">
+            Ruby on rails
+          </li>
+          <li class="skill-1">
+            css
+          </li>
+        </ul>
+        <div class="separator-line"></div>
+        <div class="display-btn">
+          <button class="btn"> See live <img src="${item.Image1}"</button>
+          <button class="btn"> See Source <img src="${item.Image2}"</button>
+        </div>
+        </div>
+        </div>
+      </div>
+      </div>
+      `).join('');
```

Comment:
- You have done well so far, but you have not styled your pop-up window according to the Figma design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
![gergr](https://github.com/jerymey/Portfolio-setup/assets/73607512/f762621a-b993-4f7d-b752-95d2e910808c)

Design template:
![thstrhsert](https://github.com/jerymey/Portfolio-setup/assets/73607512/8e722d05-bd2d-45ed-a7e5-ebde4cd8c94a)

- In the mobile version, please make sure you assign the buttons below in the centre as it is aligned in the design template. Please fix this 🙏 
Figma template:
![tethethesh](https://github.com/jerymey/Portfolio-setup/assets/73607512/3392bbfb-1bcb-4a38-9075-63269a16f5d7)
Design template:
![ehethsrt](https://github.com/jerymey/Portfolio-setup/assets/73607512/7c626489-b4e8-4b00-91fa-e8a232ee5e6e)